apps/playground/src/app/modules/components/alert-module/alert-module.component.html

```diff
@@ -1,47 +1,132 @@
-<section class="module-description">
-  <h2>Description</h2>
-  <p>Some description</p>
+<section class='module-description'>
+  <h2>DsfrAlertComponent</h2>
+  <p>
+    The <code>DsfrAlertComponent</code> purpose is to represent the alert
+  </p>
+  <p>
+    It is based on the
+    <a
+      target='_blank'
+      hreflang='fr'
+      title='The documentation page for the Alert Component in the DSFR'
+      href='https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/alerte/'>
+      Alerte
+      <mat-icon>open_in_new</mat-icon>
+    </a>
+    component in the
+    <a
+      target='_blank'
+      hreflang='fr'
+      title='The DSFR documentation'
+      href='https://www.systeme-de-design.gouv.fr/'>
+      DSFR
+      <mat-icon>open_in_new</mat-icon>
+    </a>
+    .
+  </p>
+  <p>It has several <code>@Input()</code> properties:</p>
+  <dl class='input-properties'>
+    <play-input-property-definition
+      name='title'
+      type='It takes a string'>
+      It is required when no description is provided as content inside the component's opening and closing tags.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name='type'
+      type='It takes a string with 4 possible values : <code>info</code>, <code>warning</code>, <code>success</code>, <code>error</code>'>
+      It is required.
```

Comment:
`The type will influence the general look of the alert, namely the color and the icon displayed. You can import the <code>AlertType</code> enum in your app to safely provide a value for this property.`